geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseFeatureSource.scala

```diff
@@ -189,8 +187,9 @@ class HBaseFeatureSource(entry: ContentEntry,
   }
 
   private def isBounded(temporalFilters: Seq[Filter]): Boolean = {
-    val interval = FilterHelper.extractInterval(temporalFilters, Some(sft.getDescriptor(dtgIndex).getLocalName))
-    interval != null && interval.getStartMillis != minDateTime && interval.getEndMillis != maxDateTime
+    andOption(temporalFilters)
+        .flatMap(FilterHelper.extractIntervals(_, sft.getDescriptor(dtgIndex).getLocalName).headOption)
+        .exists(i => i._1.getMillis != MinDateTime && i._2.getMillis != MaxDateTime)
```

Comment:
The changes here seem to make sense.  Have you or someone else had a chance to validate with the HBase QuickStart?